src/dwn/web5-dwn.js

```diff
@@ -53,5 +53,5 @@ class Web5DWN {
 }
 
 export {
-  Web5DWN,
+  Web5Dwn,
```

Comment:
similar here - this is how we CamelCase?

Reply:
yeah i really dislike this

Reply:
This is exactly how camel casing works in `dwn-sdk-js`

The actual class in `dwn-sdk-js` is literally `Dwn`:
https://github.com/TBD54566975/dwn-sdk-js/blob/07bcddb399d908fd431f4769dd0e48f1f03d9e68/src/dwn.ts#L28

It was completely inconsistent through Web5 JS previously.  It is now consistent and both libraries match the approach to casing.